spring-boot-project/spring-boot-docs/pom.xml

```diff
@@ -1440,6 +1440,13 @@
 					<plugin>
 						<groupId>org.asciidoctor</groupId>
 						<artifactId>asciidoctor-maven-plugin</artifactId>
+						<dependencies>
+							<dependency>
+								<groupId>io.spring.asciidoctor</groupId>
+								<artifactId>spring-asciidoctor-extensions</artifactId>
+								<version>0.1.3.RELEASE</version>
```

Comment:
FTR this would break our build as this is not on Maven Central and we don't build against `repos.spring.io`.

Reply:
Ok, I'll add`repos.spring.io` as a `<repository>`

Reply:
Done.

Reply:
It should be a `<pluginRepository>` as we've done in the Gradle plugin:

https://github.com/spring-projects/spring-boot/blob/9e640cf6efebf3efe662720da70981e93d670280/spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin/pom.xml#L184-L190

Reply:
I see, Thanks, Will revert